src/server/game/Conditions/DisableMgr.cpp

```diff
@@ -381,6 +392,7 @@ bool IsDisabledFor(DisableType type, uint32 entry, Unit const* unit, uint8 flags
         case DISABLE_TYPE_OUTDOORPVP:
         case DISABLE_TYPE_ACHIEVEMENT_CRITERIA:
         case DISABLE_TYPE_MMAP:
+        case DISABLE_TYPE_AUCTIONHOUSE_ITEM:
```

Comment:
Then, I think, will be better to implement DISABLE_TYPE_ITEM with different flags to avoid multiply Types. Give me a time and i'll implement this